### DISCUSSION TITLE
Is full Windows testing worth it?

### DISCUSSION THREAD

**Discussion by astrojuanlu** [kedro-org/kedro#4143] [Idea]
According to ClickPy, Windows accounts for 1 % of PyPI downloads in the past ~6 months

![Screenshot 2024-09-05 at 11-10-39 ClickHouse Query WITH systems AS ( SELECT system FROM pypi pypi_downloads_per_day_by_version_by_system WHERE (date {min_date String} Date32) AND (date {max_date String} Date32) AND (project {package_name String})](https://github.com/user-attachments/assets/b70dff5e-ebb2-4640-a83e-4d86abb5dd22)

**Comment by merelcht** [comment]
I would _love_ to not care about Windows, but a lot of firm colleagues have Windows machines and I feel more confident saying Kedro works on Windows because we have these tests.

**Comment by noklam** [comment]
I guess we need to be careful about how many CI is running in Linux.

**Comment by astrojuanlu** [comment]
To clarify, I'm not advocating for dropping Windows support, or stop testing. "Full" testing though (meaning: every starter, every dataset, every Python version) with these numbers seems like a stretch...
